plugins-compat-tester/src/main/java/org/jenkins/tools/test/maven/ExternalMavenRunner.java

```diff
@@ -113,4 +113,17 @@ public void run(Config config, File baseDirectory, File buildLogFile, String...
         }
     }
 
+    private Set<String> getTypes(Config config) {
+        if (config == null || config.userProperties == null || !config.userProperties.containsKey("types")) {
+            return null;
```

Comment:
Default to surefire?

Reply:
Updated to use `surefire` as default here: https://github.com/jenkinsci/plugin-compat-tester/pull/271/commits/9f198e617eb1223c4a89ba82bdb7708c4e201c0f